.github/workflows/run-tests.yml

```diff
@@ -13,6 +13,21 @@ on:
       commit:
         description: "Commit SHA"
         required: true
+      run_single_functional_test:
+        description: "Whether to run a single test. If so, the rest of the input fields are required."
+        type: boolean
+        default: false
+      n_runs:
+        description: "[Single Test Only] Number of times to repeat the single test per database type"
+        type: number
+        default: 1
+      test_name:
+        description: "[Single Test Only] Name of the test suite to run, include -testify.m flag here if desired (i.e. 'TestAcquireShard_DeadlineExceededErrorSuite' or 'TestFunctionalSuite -testify.m=TestUpdateWorkflow')"
```

Comment:
using flags like `-run TestFunctionalSuite/TestUpdateWorkflow` works, so you shouldn't need `-testify.m`

Reply:
good to know! I've been using -testify.m since first learning that method of running testify subtests